src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Square extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int side;
+    private Random random = new Random();
+
+    @Override
+    public void randomizeProperty() {
+        side = random.nextInt(MAXIMUM_SIZE);
```

Comment:
Magic numbers should be avoided. In this case, 'MAXIMUM_SIZE' is already defined, but instead of using 'nextInt(MAXIMUM_SIZE)', you should add 1 to the result to avoid generating a square with a side of 0.